tests/quic/testExample.nim

```diff
@@ -38,11 +43,16 @@ suite "examples from Readme":
       let connection = await listener.accept()
       check connection.certificates().len == 1
       let stream = await connection.incomingStream()
-      let readMessage = await stream.read()
+
+      var read: seq[byte] = @[]
+      while read.len < message.len:
+        let readMessage = await stream.read()
+        read = read & readMessage
```

Comment:
would it make sense to have variant of `stream.read(length)` that will do exactly this logic?

Reply:
Maybe? but for sure this feature already exists in nim-libp2p:
https://github.com/vacp2p/nim-libp2p/blob/master/libp2p/stream/lpstream.nim#L138-L173